contrib/util/restore

```diff
@@ -163,16 +182,16 @@ while [ $LOOKING -eq 1 ]; do
 done
 
 # Extract the OpenEMR web directory tree.
-dlg_info "Extracting $BAKDIR/openemr.tar.gz ..."
-mkdir openemr
-cd openemr
-tar zxf ../openemr.tar.gz
+dlg_info "Extracting $BAKDIR/$webroot_name.tar.gz ..."
+mkdir $webroot_name
```

Comment:
I think "openemr" is a temporary directory here, so shoudn't matter what its name is?